src/parser_data.h

```diff
@@ -184,6 +189,11 @@ struct ly_in;
 
 /** @} datavalidationoptions */
 
+/**
```

Comment:
it should be part of the `datavalidationoptions` group, shouldn't it?

Reply:
Fixed.